src/js/flow/acquia_lift.variations.views.js

```diff
@@ -45,7 +45,7 @@
        *   True if showing the highlight, false if no highlight should be shown.
        */
       highlightAnchor: function(show) {
-        var highlightClass = 'acquia-lift-page-variation-item';
```

Comment:
Yes, I have delayed all changes related to tests, and will fix all tests together later.